### PR TITLE
test: add order test

### PR DESCRIPTION
**Pull request by ConorGrocock**
## Description

Add a unit test to ensure that the pixels.json file is ordered correctly (y low to high then x low to high)

## Related issues

List any related issues here
Issue #206

### CODE REVIEW

File: _data/pixels.json [hunk 1]
```diff
@@ -301,8 +301,8 @@
     {"y": 39, "x": 7, "color": "#19d432", "username": "Dj-Denis"},
     {"y": 39, "x": 9, "color": "#ae91ff", "username": "AaronGaddes"},
     {"y": 39, "x": 10, "color": "#4BBEE3", "username": "gemanepa"},
-    {"y": 39, "x": 12, "color": "#02302f", "username": "RohanMenon"},
     {"y": 39, "x": 11, "color": "#f2de3c", "username": "PJijin"},
+    {"y": 39, "x": 12, "color": "#02302f", "username": "RohanMenon"},
```
Comment (ConorGrocock) [inline _data/pixels.json]: This change was to fix the order of pixels. In order to make the test pass.

File: __tests__/data.test.js [hunk 2]
```diff
@@ -84,4 +84,18 @@ describe('pixels', () => {
...
   });
+
+  test('every pixel should be ordered from smallest to largest in y then x', async () => {
+    const pixels = await loadJson('pixels.json');
+    var lastPixel;
+    for (const pixel of pixels.data) {
+      if (lastPixel !== undefined) {
+        // Check if the current pixel is further along in either the x
+        // or the y axis
+        expect(pixel.x > lastPixel.x || pixel.y > lastPixel.y).toBeTruthy();
```
Comment (clanghout) [inline __tests__/data.test.js]: I think your test currently does not catch every edge case. Due to lazy evaluation, whenever the x coordinate is larger, than the previous line we continue to the next pixel, without checking the y coordinate. 

I suggest changing the logic to `pixel.x > lastPixel.x && pixel.y >= lastPixel.y` to ensure both x and y coordinate are always in correct order.
